addons/controllers/packageinstallstatus_controller.go

```diff
@@ -0,0 +1,434 @@
+	// for management cluster, the naming convention for core/additional package installs is just <addon short name>
+	if clusterRole == clusterRoleWorkload {
+		pkgiName = util.GeneratePackageInstallName(clusterName, pkgRefName)
+	} else if clusterRole == clusterRoleManagement {
+		pkgiName = pkgShortname
+	}
+
+	pkgi := &kapppkgiv1alpha1.PackageInstall{}
+	objectKey := client.ObjectKey{Namespace: pkgiNamespace, Name: pkgiName}
+
+	// Note: use of the client (remote or not) depends on whether the corresponding cluster is
+	// workload or management which is determined in the Reconcile function
+	if err := c.Get(ctx, objectKey, pkgi); err != nil {
+		log.Error(err, fmt.Sprintf("unable to get PackageInstall '%s/%s'", pkgiNamespace, pkgiName))
+		return false, err
+	}
+
+	// For each package, create a single summary condition from the condition slice
+	pkgiCondition := util.SummarizeAppConditions(pkgi.Status.Conditions)
+
+	// Note: in case of encountering an unknown PackageInstall condition, just return err=nil and proceed with handling the next package
+	if pkgiCondition.Type == util.UnknownCondition {
+		log.Error(fmt.Errorf("unknown condition type for '%s/%s'", pkgi.Name, pkgi.Name), "unknown condition")
+		return false, nil
+	}
+
+	condition := runtanzuv1alpha3.Condition{
+		Type: runtanzuv1alpha3.ConditionType(strings.Title(pkgShortname)) + "-" +
+			runtanzuv1alpha3.ConditionType(pkgiCondition.Type),
+		Status:             pkgiCondition.Status,
+		UsefulErrorMessage: util.GetKappUsefulErrorMessage(pkgi.Status.UsefulErrorMessage),
+		LastTransitionTime: metav1.NewTime(time.Now().UTC().Truncate(time.Second)),
+	}
+
+	// Only add a new condition entry for the PackageInstall in the clusterBootstrapStatus in case it doesn't already exist
+	// If it does, just update it with the new condition
+	var conditionExists bool
+	var conditionChanged bool
+	for i, existingCond := range clusterBootstrapStatus.Conditions {
+		if !strings.Contains(string(existingCond.Type), strings.Title(pkgShortname)) {
+			continue
+		}
+		conditionExists = true
+		if !util.HasSameState(&clusterBootstrapStatus.Conditions[i], &condition) {
+			conditionChanged = true
+			clusterBootstrapStatus.Conditions[i] = condition
+		}
+	}
+	if !conditionExists {
+		conditionChanged = true
+		clusterBootstrapStatus.Conditions = append(clusterBootstrapStatus.Conditions, condition)
+	}
+
+	return conditionChanged, nil
+}
+
+// watchPackageInstalls sets a remote watch on the provided cluster on the Kind resource
+func (r *PackageInstallStatusReconciler) watchPackageInstalls(ctx context.Context, cluster *clusterapiv1beta1.Cluster, log logr.Logger) error {
+	// If there is no tracker, don't watch remote package installs
+	if r.Tracker == nil {
+		return nil
+	}
+
+	return r.Tracker.Watch(ctx, &remote.WatchInput{
+		Name:         "watchPackageInstallStatus",
```

Comment:
question: `watchPackageInstall` ?

Reply:
As the reconciler reacts only to ```Status``` changes in the ```PackageInstall``` resource, ```watchPackageInstallStatus``` could be a better naming, but on the other hand, one might argue that ```Status``` is only a sub-resource and can not be directly watched. 
As the name of the reconciler and controller is ```PackageInstallStatusReconciler```, I think is better to stay consistent and keep using  ```watchPackageInstallStatus```. Please let me know if you had objections to this naming.